dronesym-frontend/package.json

```diff
@@ -50,5 +50,14 @@
     "ts-node": "~3.0.4",
     "tslint": "~5.3.2",
     "typescript": "~2.3.3"
-  }
+  },
+  "description" :"Application For Controlling Drone Systems Remotely",
```

Comment:
Let's move all these to the top of the page and have it below the license

Reply:
ok i will send another Pull Request with the specified changes @malithsen

Reply:
https://github.com/scorelab/DroneSym/pull/76          <----- NEW PR